### ISSUE TITLE
same as win11 auto color management?

### ISSUE THREAD

**Issue by 24fpsDaVinci** [ledoge/novideo_srgb#70]
is this the same as auto color management ACM in win11 but with gamma correction and dithering, and works in 3D applications?

**Comment by ValeZAA** [comment]
I think you are overestimating how automatic 22H2 Moments 3 actually is. Yes, the newer updates added HDR JXR wallpapers – it is as cool as Chrome HDR support and MacOS HDR support – but it far from over.

https://blogs.windows.com/windows-insider/2023/08/04/announcing-windows-11-insider-preview-build-25921/

**Comment by Tallulah88** [comment]
> I think you are overestimating how automatic 22H2 Moment 3 actually is.
I believe he's talking about the "Auto color management" (ACM) feature that is still only available on the Dev channel. I didn't try it out but it looks promising. You can find some information about it by searching "36371531" (that's the code to activate the feature on the Dev channel). I'm attaching a screenshot that I found online.
![Build-25309-Auto-Color-Management-setting-in-the-Advanced-display-settings-page](https://github.com/ledoge/novideo_srgb/assets/104866331/eeeb3bad-c8e0-4248-83c6-23f7465cef0f)

**Comment by 24fpsDaVinci** [comment]
yes Im referring to "Auto color management" (ACM), one limitation is if the edid contains an HDR block (most faux hdr monitors) ACM will be disabled.

**Comment by Tallulah88** [comment]
Windows 11 has introduced an updated ‘Color Management’ feature in its Developer Build 26058, which I’ve been testing. Unfortunately, it doesn’t seem to be great. I’m not sure if I’m missing something, but it appears to ignore gamma correction entirely, possibly focusing solely on gamut adjustment. On a positive note, the dithering performance is good. For those interested, I’m attaching a comparison: my display’s default settings (without calibration or profiling), versus Windows 11 Color Management, versus novideo_srgb.

[no_color_management.zip](https://github.com/ledoge/novideo_srgb/files/14403256/no_color_management.zip)
[win11_ACM.zip](https://github.com/ledoge/novideo_srgb/files/14403257/win11_ACM.zip)
[novideo_srgb.zip](https://github.com/ledoge/novideo_srgb/files/14403258/novideo_srgb.zip)

The profile was made with DisplayCAL's sRGB preset with these modifications:
- Calibration speed: Very high
- Profile type: Curves + Matrix
- Black point compensation disabled
- Testchart: custom with 256 "Neutral" patches

Verification settings:
- Simulation profile: sRGB
- Use simulation profile as display profile
- Apply black output offset (100%)

**Comment by 24fpsDaVinci** [comment]
no gamma correction in windows 11 ACM, is this related to the profile not being ACM compatible?

**Comment by 24fpsDaVinci** [comment]
no gamma correction in windows 11 ACM, is this related to the profile not being ACM compatible?
there is a tool to generate ACM compatible profiles.
https://github.com/dantmnf/MHC2

**Comment by Tallulah88** [comment]
> no gamma correction in windows 11 ACM, is this related to the profile not being ACM compatible? there is a tool to generate ACM compatible profiles. https://github.com/dantmnf/MHC2

You are right, that was the problem! I had to use that tool and convert the profile using the "sdr-acm" command. Now it's working quite nicely, with great results! I noticed the results are perfect using DisplayCAL's default presets; there's no need to customize them. Although it's still a bit buggy—sometimes, if you disconnect another display, you have to disable and enable the color management again to make it work. When using Windows' new color management, Photoshop seems to be able to display colors beyond sRGB (when working with an image that is wide gamut). However, other software like XnView or Firefox doesn't seem to be able to display correctly content tagged as wide gamut. That's not an issue for me, but I wonder if they will add support for this in the future.

EDIT: Photoshop is using the wrong gamma curve now. Looks like the gamma correction is being applied twice when using Photoshop. However, Window's "Photos" image viewer works great and is also able to show wide gamut images with proper color management.

EDIT2: After a lot of testing I went back to novideo_sRGB. The Windows 11 ACM is not good enough yet.
